simulator/fetch/fetch.cpp

```diff
@@ -108,11 +108,11 @@ void Fetch<ISA>::clock_instr_cache( Cycle cycle)
 template <typename ISA>
 void Fetch<ISA>::ignore( Cycle cycle)
 {
-    /* ignore PC from other ports in the case of cache miss */ 
+    /* ignore PC from other ports in the case of cache miss */
     rp_external_target->ignore( cycle);
     rp_hold_pc->ignore( cycle);
     rp_target->ignore( cycle);
-
+    rp_stall->ignore( cycle);
```

Comment:
Everything is okay, but this string is not needed, I think.

Reply:
Well, it is the only string we would like to review with you :-)

Please check #282 for details. There is a data hazard which stalls instruction in the end of cache line, while next line has a cache miss.

You may reproduce it by running the latest version of `tt.core.s` trace.

Reply:
Ok, I will look at it more carefully.

Reply:
Thanks. We are looking forward for your decision.

Reply:
Sorry, this string is important, my bad(